### PR TITLE
add example doc page

### PR DESCRIPTION
**Pull request by wichniowski**
# Purpose of PR

Hey for our docs marathon I added a example template it should follow @domarku outline. Some things are optional though. Please let me know if we can improve this!

### CODE REVIEW

File: packages/forma-36-website/src/pages/components/example/index.mdx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+<React.Fragment>
+  <Button buttonType="primary">Primary</Button>
+  <Button buttonType="negative">Negative</Button>
+  <Button buttonType="positive">Positive</Button>
+  <Button buttonType="muted">Muted</Button>
+  <Button buttonType="naked">Naked</Button>
+</React.Fragment>
+```
+
+### With icon (`icon`)
```
Comment (m10l) [inline packages/forma-36-website/src/pages/components/example/index.mdx]:
```suggestion
### With icon (`icon`) <a name="button-with-icon" />
```

File: packages/forma-36-website/src/pages/components/example/index.mdx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  <Button buttonType="muted" icon="Settings">
+    Muted
+  </Button>
+  <Button buttonType="naked" icon="Settings">
+    Naked
+  </Button>
+</React.Fragment>
+```
+
+### Sizes (`size`)
```
Comment (m10l) [inline packages/forma-36-website/src/pages/components/example/index.mdx]:
```suggestion
### Sizes (`size`) <a name="button-sizes" />
```

File: packages/forma-36-website/src/pages/components/example/index.mdx [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+</React.Fragment>
+```
+
+## Table of contents
+
+1. [Best practices](#best-practices)
+2. [Variations](#variations)
+   1. [Button types](#button-types)
+3. [Copy considerations](#copy-considerations)
+4. [Accessibility](#accessibility)
```
Comment (sarakalinoski) [inline packages/forma-36-website/src/pages/components/example/index.mdx]: Accessibility guidelines

File: packages/forma-36-website/src/pages/components/example/index.mdx [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+  <Button buttonType="primary">Primary</Button>
+</React.Fragment>
+```
+
+## Table of contents
+
+1. [Best practices](#best-practices)
+2. [Variations](#variations)
+   1. [Button types](#button-types)
+3. [Copy considerations](#copy-considerations)
```
Comment (sarakalinoski) [inline packages/forma-36-website/src/pages/components/example/index.mdx]: Writing guidelines

File: packages/forma-36-website/src/pages/components/example/index.mdx [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+```
+
+## Copy considerations <a name="copy-considerations" />
+
+(optional) If the component has specific copy considerations mention them in this section
+
+- Do not use punctuation following a label
+- Use sentence-style caps (in which only the first word of a sentence or term is capitalized) on labels
+
+## Accessibility <a name="accessibility" />
```
Comment (sarakalinoski) [inline packages/forma-36-website/src/pages/components/example/index.mdx]: Accessibility guidelines

File: packages/forma-36-website/src/pages/components/example/index.mdx [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+    Large
+  </Button>
+  <Button buttonType="muted">Default</Button>
+  <Button buttonType="muted" size="small">
+    Small
+  </Button>
+</React.Fragment>
+```
+
+## Copy considerations <a name="copy-considerations" />
```
Comment (sarakalinoski) [inline packages/forma-36-website/src/pages/components/example/index.mdx]: Writing guidelines

File: packages/forma-36-website/src/pages/components/example/index.mdx [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+Add best practices for usage in the UI, think about copy and accessibility or general UX guidelines
+E.g (Button Component):
+
+- Position buttons consistently in the interface
+- Reduce complexity by using a small number of actions. Too many actions can create confusion.
+
+## Variations <a name="variations" />
+
+Describe important properties on the component and give examples. Important properties configure a component to serve its main purposes
```
Comment (sarakalinoski) [inline packages/forma-36-website/src/pages/components/example/index.mdx]: Describe alternative ways the component may be configured depending on varying use cases.

File: packages/forma-36-website/src/pages/components/example/index.mdx [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+  <Button buttonType="muted">Default</Button>
+  <Button buttonType="muted" size="small">
+    Small
+  </Button>
+</React.Fragment>
+```
+
+## Copy considerations <a name="copy-considerations" />
+
+(optional) If the component has specific copy considerations mention them in this section
```
Comment (sarakalinoski) [inline packages/forma-36-website/src/pages/components/example/index.mdx]: If the component is configured with text, share writing guidelines to ensure consistency. (Optional)

File: packages/forma-36-website/src/pages/components/example/index.mdx [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+
+- Do not use punctuation following a label
+- Use sentence-style caps (in which only the first word of a sentence or term is capitalized) on labels
+
+## Accessibility <a name="accessibility" />
+
+(optional) If the component has specific a11y requirements mention them here
```
Comment (sarakalinoski) [inline packages/forma-36-website/src/pages/components/example/index.mdx]: If it is possible to add accessibility improvements to the component, share accessibility guidelines to ensure consistency. (Optional)

File: packages/forma-36-website/src/pages/components/example/index.mdx [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+    Small
+  </Button>
+</React.Fragment>
+```
+
+## Copy considerations <a name="copy-considerations" />
+
+(optional) If the component has specific copy considerations mention them in this section
+
+- Do not use punctuation following a label
```
Comment (sarakalinoski) [inline packages/forma-36-website/src/pages/components/example/index.mdx]: E.g.: 
- Do not use punctuation following a label
- Use sentence-style caps (in which only the first word of a sentence or term is capitalized) on labels

File: packages/forma-36-website/src/pages/components/example/index.mdx [hunk 11]
```diff
@@ -0,0 +1,100 @@
...
+- Position buttons consistently in the interface
+- Reduce complexity by using a small number of actions. Too many actions can create confusion.
+
+## Variations <a name="variations" />
+
+Describe important properties on the component and give examples. Important properties configure a component to serve its main purposes
+
+**_E.g: Button Component_**
+
+The Button component can be configured in a number of different ways. Here is a guide to when to use certain variations.
```
Comment (sarakalinoski) [inline packages/forma-36-website/src/pages/components/example/index.mdx]: The Button component can be configured in a number of different ways. Follow this guidance to determine when to use certain variations:

File: packages/forma-36-website/src/pages/components/example/index.mdx [hunk 12]
```diff
@@ -0,0 +1,100 @@
...
+
+1. [Best practices](#best-practices)
+2. [Variations](#variations)
+   1. [Button types](#button-types)
+3. [Copy considerations](#copy-considerations)
+4. [Accessibility](#accessibility)
+
+## Best practices <a name="best-practices" />
+
+Add best practices for usage in the UI, think about copy and accessibility or general UX guidelines
```
Comment (sarakalinoski) [inline packages/forma-36-website/src/pages/components/example/index.mdx]: Include best practices for how the component should be used in the context of the overall experience.

File: packages/forma-36-website/src/pages/components/example/index.mdx [hunk 13]
```diff
@@ -0,0 +1,100 @@
+---
+title: 'Example Component'
+type: 'component'
+github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/FormLabel'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/components-formlabel--default'
+---
+
+Add a description about the component and its function in the user interface here
```
Comment (sarakalinoski) [inline packages/forma-36-website/src/pages/components/example/index.mdx]: Introduce the component with a description about the component and its function within the user interface.

File: packages/forma-36-website/src/pages/components/example/index.mdx [hunk 14]
```diff
@@ -0,0 +1,100 @@
...
+E.g: TextLinks communicate actions and highlight linked resources to users.
+
+Add an example of the component like this:
```
Comment (sarakalinoski) [inline packages/forma-36-website/src/pages/components/example/index.mdx]: To give the user a preview of the component, include an example rendering.

**Pull request closed without merging**